inc/wpseo-non-ajax-functions.php

```diff
@@ -385,8 +385,10 @@ function wpseo_adminbar_score( $score ) {
  * Enqueue CSS to format the Yoast SEO adminbar item.
  */
 function wpseo_admin_bar_style() {
+	
```

Comment:
There seem to be some spaces on this line.
Could you remove these?

We have fairly strict Coding Standard checking, which is failing on this problem.